src/test/resources/elastic-stack.yml

```diff
@@ -23,7 +23,7 @@ services:
       - "action.destructive_requires_name=false"
     image: docker.elastic.co/elasticsearch/elasticsearch:${ELASTIC_STACK_VERSION:-9.0.0}
     ports:
-      - "9200"
+      - "9200:9200"
```

Comment:
I'd keep the port random to avoid issues

Reply:
@jgreffe The Docker containers should be able to talk to each other using the non-randomized container ports (unless Kibana needs to know its own user-facing URL similar to `JenkinsLocationConfiguration` or MinIO's `MINIO_BROWSER_REDIRECT_URL`). From a quick check, randomized port mappings from the Jenkins side works ok with the test, we just have to revert the changes to `ElasticStack.getEsUrl`, `.getKibanaUrl`, and `.getFleetUrl`, and then until https://github.com/jenkinsci/opentelemetry-plugin/issues/1088 is fixed there are some warnings in the logs until the test reconfigures things to point to the randomized ports.